### PR TITLE
Lineage 18.1 update

### PR DESCRIPTION
**Pull request by linckandrea**

### CODE REVIEW

**Comment by linckandrea** [comment]
hold

File: BoardConfigPlatform.mk [hunk 1]
```diff
@@ -137,6 +140,7 @@ endif
 OVERRIDE_RS_DRIVER := libRSDriver_adreno.so
 
 ### RIL
+DEFAULT_APNS_FILE := $(PLATFORM_PATH)/config/telephony/apns-conf.xml
```
Comment (Flamefire) [inline BoardConfigPlatform.mk:143]: Please remove this until LOS fixed their script. See https://github.com/whatawurst/android_device_sony_yoshino-common/issues/46#issuecomment-857421284 for details

**Review by Flamefire — CHANGES_REQUESTED** [review]

**Comment by derfelot** [comment]
Any updates? If not, I will pick the powerhint commit to an update I want to do later and leave the APN stuff now.

**Comment by Flamefire** [comment]
I'd remove the 2nd commit and keep the other 2. Tested on my phone with 17.1 and working fine so far.

**Comment by derfelot** [comment]
I'll close this and pick the two working ones.

**Pull request closed without merging**